packages/react-components/src/components/SendBox.tsx

```diff
@@ -13,6 +13,7 @@ import {
 } from './styles/SendBox.styles';
 import { BaseCustomStylesProps } from '../types';
 import { isDarkThemed } from '../theming/themeUtils';
+import { COMPONENT_UI_IDS } from './identifiers';
```

Comment:
nit: Use CamelCase for constants.

Reply:
Any particular reason?

Reply:
Hmm.. Ignore this recommendation. Looks like we don't have a style enforcement for this. Nor does Azure: https://github.com/Azure/azure-sdk-for-js/blob/e823b26f4600a698eb1c06f48da9b780fb2331a7/common/tools/eslint-plugin-azure-sdk/src/configs/azure-sdk-base.ts#L67

In JavaScript, top level definitions are all constants anyway (most functions are also just `const` function objects).
So I thought that there might be a convention to not capitalize string / numerical "constants" either ('cause what's so different about them?)

A quick look around tells me that UPPER_CASE is indeed the convention though, so you're right:

* [Airbnb](https://github.com/airbnb/javascript/blob/master/README.md#naming--uppercase) uses UPPER_CASE, but the style guide explicitly says that it should be used for exported constants that are guaranteed to not change (in case of nested object values). This is what I was afraid of...
* https://google.github.io/styleguide/jsguide.html#naming-constant-names also uses UPPER_CASE
* https://github.com/rwaldron/idiomatic.js/ also uses UPPER_CASE